cli/src/CliArgsParser.cpp

```diff
@@ -248,23 +249,38 @@ void OpenSpaceNetArgs::parseArgsAndProcess(int argc, const char* const* argv)
         validateArgs();
     } catch (...) {
         if (displayHelp) {
-            printUsage(action);
+            printUsage(osnArgs.action);
         }
         throw;
     }
 
     if (displayHelp) {
-        printUsage(action);
+        printUsage(osnArgs.action);
         return;
     }
 
     setupLogging();
 
-    OpenSpaceNet osn(*this);
+
+}
+
+void CliArgsParser::startOSNProcessing()
+{
+    OpenSpaceNet osn(osnArgs);
+    if (osnArgs.action == Action::LANDCOVER) {
+        std::vector<ProgressCategory> cats = {ProgressCategory("Loading", "Loading the image"),ProgressCategory("Classifying", "Classifying the image")};
```

Comment:
Landcover progress in OSN master used this: `MultiProgressDisplay progressDisplay({ "Loading", "Classifying" });` So I was just trying to be consistent with that. 

I can change it to reading if we want to be consistent between landcover and detection